master/coverage.py

```diff
@@ -116,16 +107,21 @@
     ),
 
     # Run Julia, gathering coverage statistics
+    steps.MakeDirectory(dir=util.Interpolate("mkdir %(prop:juliadir)s/LCOV")),
     steps.ShellCommand(
-        name="Run inlined tests",
-        command=[util.Interpolate("%(prop:juliadir)s/bin/julia"), "--sysimage-native-code=no", "--code-coverage=all", "-e", run_coverage_cmd],
+        name="Run tests",
+        command=[util.Interpolate("%(prop:juliadir)s/bin/julia"),
+                 "--sysimage-native-code=no", util.Interpolate("--code-coverage=%(prop:juliadir)s/LCOV/cov-%%p.info"),
+                 "-e", run_coverage_cmd],
         timeout=3600,
     ),
-    steps.ShellCommand(
-        name="Run non-inlined tests",
-        command=[util.Interpolate("%(prop:juliadir)s/bin/julia"), "--sysimage-native-code=no", "--code-coverage=all", "--inline=no", "-e", run_coverage_cmd],
-        timeout=7200,
-    ),
+    #steps.ShellCommand(
+    #    name="Run non-inlined tests",
+    #    command=[util.Interpolate("%(prop:juliadir)s/bin/julia"),
+    #             "--sysimage-native-code=no", util.Interpolate("--code-coverage=%(prop:juliadir)s/LCOV/cov-%%p.info"), "--inline=no",
+    #             "-e", run_coverage_cmd],
+    #    timeout=7200,
+    #),
```

Comment:
Do we no longer have non-inlined tests?

Reply:
We haven't in a couple years, although mainly I was just selfishly interested in getting results from the buildbot faster.

Reply:
Sounds good to me, I'll just remove this chunk completely then after merging.